streams/src/main/java/org/apache/kafka/streams/processor/internals/AssignedStreamsTasks.java

```diff
@@ -189,10 +201,13 @@ RuntimeException closeRestoringTasks(final Set<TaskId> restoringTasksToClose,
     }
 
     private RuntimeException closeRunning(final boolean isZombie,
-                                          final StreamTask task,
-                                          final List<TopicPartition> closedTaskChangelogs) {
+                                          final StreamTask task) {
         removeTaskFromRunning(task);
-        closedTaskChangelogs.addAll(task.changelogPartitions());
```

Comment:
Why did we remove this line here from `closeRunning` but we have the same call in `suspendRunningTasks`.  More of a question for my own education, I wouldn't hold up merging for this.

Reply:
We no longer need to keep track of which changelog partitions were lost after a `onPartitionsLost` since we just clear everything. `#closeRunning` is actually only called on zombie tasks, so we can just remove it entirely here (non-zombie running tasks are closed by first suspending and then closing as suspended)